test/functional/test_framework/test_node.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, i, datadir, *, chain, rpchost, timewait, bitcoind, bitcoin_cl
         self.stderr_dir = os.path.join(self.datadir, "stderr")
         self.chain = chain
         self.rpchost = rpchost
-        self.rpc_timeout = timewait
+        self.rpc_timeout = int(timewait)
```

Comment:
why is this needed, when authproxy does the casting already?

Reply:
In some previous variant of my code I encountered such a problem. Maybe just because I was still writing it. Right now, it seems that all those conversions aren't needed anymore. I'm just executing some tests and they're all passing.